doc/classes/ParticlesMaterial.xml

```diff
@@ -266,6 +281,36 @@
 		<member name="tangential_accel_min" type="float" setter="set_param_min" getter="get_param_min" default="0.0">
 			Minimum tangential acceleration.
 		</member>
+		<member name="turbulence_active" type="bool" setter="set_turbulence_active" getter="get_turbulence_active" default="false">
+			Enables and disables Turbulence for the particle system.
+		</member>
+		<member name="turbulence_influence_max" type="float" setter="set_param_max" getter="get_param_max">
+			Minimum turbulence influence on each particle.
```

Comment:
```suggestion
			Minimum turbulence influence on each particle. The actual amount of turbulence influence on each particle is calculated as a random value between [member turbulence_influence_min] and [member turbulence_influence_max] and multiplied by the amount of turbulence influence from [member turbulence_influence_over_life].
```

Reply:
Thanks for the additional documentation! I'll add it in the next update!